### DISCUSSION TITLE
Replace feed with something useful

### DISCUSSION THREAD

**Discussion by woylie** [community/community#37464] [Other Feature Feedback, Questions, & Ideas]
The new feed contains 100% useless information. Github is not a social media site. Don't distract your users from their actual intentions. Please use the home page for useful information, e.g. an overview over my current issues, requested pull request reviews etc.

**Comment by Sergio0694** [comment]
+1 to this. I will also add: why does it show repositories that **other users I don't even follow have starred**?? Why would I care about that or have that spam the home feed? Half my feed is just "X added repository Y to list Z", where I don't follow X...? 🤦‍♂️

**Comment by queenofcorgis** [comment]
Thanks for the feedback! Could you provide additional context for what you would want to see on the homepage feed?

**Reply by Sergio0694** [reply]
> "what you would want to see on the homepage feed?"

I was perfectly content with exactly what I had before, ie. new stars/forks/follows for my user and my own repositories. If I really _had_ to have more stuff, the only thing that would personally be useful to me would be to also see new releases _from repositories I follow_. Everything else (eg. activity from random people), please no, it's just noise 😅

I will also add, I genuinely don't understand who could possibly want to see that "person X (that you don't even follow) starred repo Y". Like... Why should anyone care about activity from random people, let alone want that in your personal feed 🤷‍♂️

**Reply by luni3359** [reply]
I know you're forced to ask the obvious due to policy but unfortunately this update makes no sense unless you guys are trying to turn into a social media website, which github isn't (or wasn't). You would have to rollback the entire update because it provides no value to users to see pointless activity every time I open the feed, and the only option to fight this is to manually mark off everybody showing up. There's a reason why the follow button exists.

**Comment by woylie** [comment]
Things I care about when I open the dashboard in descending order of importance:

1. pull reviews requested
2. issues I'm currently assigned to
3. repositories I frequently work on
4. activity feed
5. my teams

As for 1 and 2, those could be additional widgets.

Things I care about in the activity feed in descending order of importance

1. opened/closed issues and pull requests in my own repositories
2. opened/closed issues and pull requests in the repositories of my teams
3. Any activities in repositories I'm watching _depending on the watch settings_
4. new repositories created in my teams
5. new releases in the repositories of my teams
6. new releases of repositories I starred
7. someone starring one of my own repositories
8. activities of people I follow (I don't follow anybody, but for the sake of argument)

Things I _do not_ care about in my feed:

1. Any activities of people I don't follow.
2. Any activities of repositories that are not my own or of my team and that I did not star and do not watch.

I would be grateful if Github was more considerate of its user's focus and attention. We come here to get work done, after all. At the very least, give your users a choice when it comes to recommendations. Any explore/discover kind of information should be optional. At the moment, the feed is only distracting and I had to disable it. That's a lot of squandered space on the dashboard.

<img width="718" alt="image" src="https://user-images.githubusercontent.com/13847569/198736219-0384b544-a01d-4f59-b65d-0fa3759cbdad.png">

**Comment by ghost** [comment]
Github is turning into a facecode or codebook. Horrible!

**Comment by DmitriiShamrikov** [comment]
The new feed is absolutley disgusting. Why does every website have to turn into facebook clone nowadays?

**Comment by jfullstackdev** [comment]
hey guys, while you are waiting for that personal dashboard, you can start creating an Org, that way, you will have a personalized dashboard for all the repos within, it's highly focused because it's all about the feeds within that repo and creating an Org does not cause any harm, it will actually encourage you to organize the repos

**Comment by luni3359** [comment]
Please revert the entire update, it's truly pointless because the way it worked previously was objectively not distracting and productive, now I have to waste my time marking off dozens of people that I don't know about. Github is primarily about open source of course there's going to be a lot of people working on the same project you *care* about but that doesn't mean you care about the activity of absolute strangers.

**Reply by y0nei** [reply]
based

**Comment by mohkale** [comment]
I hate this update as well and am gonna speed up the plans to migrate my repos off github to gitlab. For reference however I think it's worth mentioning you should be able to disable the unwanted suggestions from the filter dropdown at the top of the feed. Whether that's saved to your account or has to be done after each sign-in I'm not sure. Either way I think it's major r/assholedesign to add a feature like this which no-one asked for and then enable it by default.

Edit: So turns out that's not enough. Even if you disable suggestions my feed still contains star updates or random notifications from people I've never interacted with. 100% worse than I expected :-(. Like I starred Linux. Now I get a notification when some random person also stars linux. This is so dumb.

**Comment by Naville** [comment]
This is just Microsoft UX at its finest
